pkg/tests/conftest.py

```diff
@@ -364,6 +421,15 @@ def salt_minion(salt_factories, salt_master, install_salt):
         overrides=config_overrides,
         defaults=config_defaults,
     )
+    # Salt factories calls salt.utils.verify.verify_env
+    # which sets root perms on /srv/salt and /srv/pillar since we are running
+    # the test suite as root, but we want to run Salt master as salt
+    if not platform.is_windows() or not platform.is_darwin():
```

Comment:
```suggestion
    if not platform.is_windows() and not platform.is_darwin():
```
We only want this code to run on Linux, right?